### PR TITLE
Chore: fix nginx image building

### PR DESCRIPTION
**Pull request by MrPink**

### CODE REVIEW

File: docker-compose-nginx.yml [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    image: aether-kernel-nginx
+    build: ./aether-kernel/nginx
+
+  # ---------------------------------
+  # ODK module
+  # ---------------------------------
+
+  odk-nginx:
+    image: aether-odk-nginx
+    build: ./aether-kernel/nginx
```
Comment (obdulia-losantos) [inline docker-compose-nginx.yml]: This should be `/aether-odk-module/nginx`

File: scripts/deploy.sh [hunk 2]
```diff
@@ -19,12 +19,13 @@ do
   for APP in "${APPS[@]}"
   do
     AETHER_APP="${PREFIX}-${APP}"
-    docker-compose build $APP
     # build nginx containers
-    docker build -t "${IMAGE_REPO}/${AETHER_APP}-nginx-${ENV}:latest" "aether-${APP}/nginx"
+    docker-compose -f docker-compose-nginx.yml build $APP-nginx
```
Comment (obdulia-losantos) [inline scripts/deploy.sh:23]: `docker-compose -f docker-compose-nginx.yml build "${AETHER_APP}-nginx" ` ?

Comment (MrPink) [reply scripts/deploy.sh:23]: it's referenced `odk-nginx` in the docker-compose file

File: docker-compose-nginx.yml (hunk 1 above)
Comment (MrPink) [reply docker-compose-nginx.yml]: 😓

File: scripts/deploy.sh (hunk 2 above)
Comment (obdulia-losantos) [reply scripts/deploy.sh:23]: :+1: Right!

**Review by obdulia-losantos — APPROVED** [review]

File: docker-compose-nginx.yml [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+  # ---------------------------------
+  # ODK module
+  # ---------------------------------
+
+  odk-nginx:
+    image: aether-odk-nginx
+    build: ./aether-odk-module/nginx
+
```
Comment (obdulia-losantos) [inline docker-compose-nginx.yml:24]: In the future we'll need the couchdb-sync one :crystal_ball:

**Pull request merged**